addons/mail/static/src/new/core/messaging_service.js

```diff
@@ -404,6 +404,9 @@ export class Messaging {
         if (channel) {
             this.threadService.update(channel, { serverData: { last_interest_dt } });
         }
+        if (channel?.type === "chat") {
```

Comment:
I believe this also allies to type group?

Reply:
and to livechat

Reply:
livechat already covered by override in `im_livechat`